docs/readmes/proposals/p005_call_tracing.md

```diff
@@ -0,0 +1,283 @@
+---
+id: p005_call_tracing
+title: Call Tracing for Troubleshooting
+hide_title: true
+---
+
+# Call Tracing for Troubleshooting
+
+- Feature owner: `@andreilee`
+- Feedback requested from: `@apad`, `@xjtian`, `@karthiksubraveti`
+
+## Summary
+
+As a general purpose debugging tool, a tracing tool will be added for
+understanding control messaging flow through a Magma access gateway.
+
+## Motivation
+
+Example use cases include the following:
+
+**Subscriber Tracing**
+
+When issues are experienced by a subscriber on the network, network operators
+would like to have a tool which allows them to see the messages between the UE,
+eNodeB, and Magma components relating to the subscriber. This feature already
+exists for other NMS tools. Ideally, this capture should be a packet capture to
+fit with existing workflows using Wireshark. Wireshark has features to decode
+and display relevant messages for Gx, Gy, S1AP, TR-069, etc., as well as
+plugins to support decoding and viewing gRPC messages which are passed between
+Magma gateway components.
+
+**Protocol Tracing**
+
+Operators would like to be able to capture traces for specific protocol stacks,
+such as SCTP or DIAMETER. This could be used for troubleshooting specific
+integrations.
+
+## Goals
+
+- Allow network operators to trace the control plane messaging for a subscriber
+- Allow network operators to trace control plane messaging for a specific
+protocol or interface
+
+## Additional Criteria
+
+These additional criteria are ideal to meet but are not necessary:
```

Comment:
It will be nice if in phase 1, we can atleast implement a super simple interface in NMS, with just basic options of start, stop and downloading the pcap. Going to swagger and doing this is additional complexity which might be nice to avoid.